### ISSUE TITLE
Apps Not Showing on Latest Android Auto (Premium License)

### ISSUE THREAD

**Issue by shoxter** [shmykelsa/AAAD#251]
My Android Auto forcibly updated within the last couple of weeks and I cannot get any of my apps to show on Android Auto anymore. I've tried reinstalling through AAAD and no luck. I don't see anything about this in the readme or the telegram so I'm trying to figure out if this is a result of the repository not being actively maintained or my issue being unique to me.

Thanks in advance!

**Comment by feliperugai** [comment]
Same thing here.

**Comment by encikjoe7** [comment]
Same here. Pixel 7 with android 14

**Comment by Swaktor** [comment]
Same to me , I am on banana droid 13, beyondx device with omni nikgapps. Android auto is systemized

**Comment by shoxter** [comment]
It's a little slimy to me that there's a paid version of this app that doesn't actually deliver on what it's supposed to do and there's no word from the person reaping those payments on whether or not it'll be fixed or why you're able to even purchase premium when it doesn't work anymore.

**Comment by gvnburak** [comment]
I used to use the free version and ot was working on my car, then i  purchased the pro version and it stopped working. Please help, dont care about the money, but app was wonderful , would love to continue to use.

**Comment by mauringo** [comment]
Same Here! So bad :(

**Comment by mauringo** [comment]
Hello, I got them working.. someway!
1- i installed any auto audio: [ https://forum.xda-developers.com/t/app-5-1-alpha-anyautoaudio-use-any-audio-app-in-auto.3607916/page-16](url)
2- enabled dev mode on Android Auto
3- bypass restricted settings on AAAD AA and AnyAutoAudio: [https://droidwin.com/android-13-restricted-settings-for-sideloaded-apps-how-to-bypass/](url)

Maybe just 2-3 is enough

**Comment by Swaktor** [comment]
> Hello, I got them working.. someway!
> 
> i have an Oneplus 9 pro full stock with android 13
> 
> 1- i installed any auto audio: [ https://forum.xda-developers.com/t/app-5-1-alpha-anyautoaudio-use-any-audio-app-in-auto.3607916/page-16](url)
> 2- enabled dev mode on Android Auto
> 3- bypass restricted settings on AAAD AA and AnyAutoAudio: [https://droidwin.com/android-13-restricted-settings-for-sideloaded-apps-how-to-bypass/](url)
> 
> Maybe just 2-3 is enough, if someone 

Works now for me ! dev mode solved issue

**Comment by gvnburak** [comment]
Any updates on this for pixel devices?

**Comment by encikjoe7** [comment]
> Any updates on this for pixel devices?

Still not working

**Comment by mdeguzis** [comment]
Same, and I just bought this before I saw the thread :(

**Comment by gvnburak** [comment]
Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices :
Uninstall all apps downloaded from AAAD.
install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13-
Reinstall desired app from AAAD and.....
Enjoy 

Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch.
June 1 google play update

**Comment by encikjoe7** [comment]
> Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices : Uninstall all apps downloaded from AAAD. install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13- Reinstall desired app from AAAD and..... Enjoy
> 
> Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch. June 1 google play update

This method not working for pixel with android 14 beta

**Comment by gvnburak** [comment]
> > Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices : Uninstall all apps downloaded from AAAD. install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13- Reinstall desired app from AAAD and..... Enjoy
> > 
> > Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch. June 1 google play update
> 
> This method not working for pixel with android 14 beta

Did you try this one. https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-14-release/

**Comment by mdeguzis** [comment]
> > > Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices : Uninstall all apps downloaded from AAAD. install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13- Reinstall desired app from AAAD and..... Enjoy
> > > Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch. June 1 google play update
> > 
> > 
> > This method not working for pixel with android 14 beta
> 
> Did you try this one. https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-14-release/

This totally worked! I'm on Android 13, but that page has a link for that. I uninstalled AAAD and all it's apps, did the installer for Google on that page, then reinstalled everything.  Awesome!

**Comment by encikjoe7** [comment]
> > > Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices : Uninstall all apps downloaded from AAAD. install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13- Reinstall desired app from AAAD and..... Enjoy
> > > Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch. June 1 google play update
> > 
> > 
> > This method not working for pixel with android 14 beta
> 
> Did you try this one. https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-14-release/

Yes. But also not working. Aaad keep stop working. And installed app not showing on android auto.

**Comment by gvnburak** [comment]
> > > > Hey pixel users ,I was lazy and didn't check the main thread. Here is the fix for pixel devices : Uninstall all apps downloaded from AAAD. install this app https://github.com/shmykelsa/AAAD/wiki/Fix-for-Pixel-Android-13- Reinstall desired app from AAAD and..... Enjoy
> > > > Tested on 2023 Nissan Pathfinder Android auto with Android 13 August 5th security patch. June 1 google play update
> > > 
> > > 
> > > This method not working for pixel with android 14 beta
> > 
> > Did you try this one. https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-14-release/
> 
> Yes. But also not working. Aaad keep stop working. And installed app not showing on android auto. 

Thanks for the feedback ,so it's better to stay away from Android 14 until we have a verified solution for AAAD

**Comment by yeahforbes** [comment]
Got an update from the developer. There seem to be about 4 duplicate issues on this, so I'll just drop a link: https://github.com/shmykelsa/AAAD/issues/269#issuecomment-1821844148
